common/rawaccel-settings.h

```diff
@@ -23,6 +23,7 @@ namespace rawaccel {
         vec2d sens = { 1, 1 };
         vec2d dir_multipliers = {};
         milliseconds time_min = DEFAULT_TIME_MIN;
+        wchar_t device_hw_id[512] = {0};
```

Comment:
size should be based on `MAX_DEVICE_ID_LEN` and `REGSTR_VAL_MAX_HCID_LEN`

from what i can tell these are only defined in usermode headers (as 200 and 1024 respectively), so we can just add the macros to this file

ref: https://docs.microsoft.com/en-us/windows-hardware/drivers/install/hardware-ids